### PR TITLE
PCVT for retrieving certificates from all platforms

### PR DESCRIPTION
**Pull request by preranam05**
The primary focus of this commit is to retrieve certificates from the servers. This enhancement facilitates support for different servers to ensure the comprehensive server integrity verification.

### CODE REVIEW

File: src/main/java/hwManifestGen/AllComponents.java [hunk 1]
```diff
@@ -195,6 +235,276 @@ public static void setOptions(Options options) {
...
+				    System.out.println("Failed to create directory: " + value);
+				}
+			    }
+
+			    // Set the file path to the specified directory with the filename "output2.pem"
+			    file = value + "output2.pem";
+			} else {
+			    System.out.println("Output Path option not provided");
+			    // Set the default output file path
+			    String value = "/home/prerana/Desktop/PCVT/PCVT/src/main/java/hwManifestGen/";
```
Comment (gustavobbrand) [inline src/main/java/hwManifestGen/AllComponents.java]: At the instructions the paths suggested were in general /opt/hpe/scl/ or /opt/hpe/scl/certificates for the outputs... so I suggest to modify this for /opt/hpe/scl/ or a directory whithin this path.

File: src/main/java/hwManifestGen/AllComponents.java [hunk 2]
```diff
@@ -195,6 +235,276 @@ public static void setOptions(Options options) {
...
+				    System.out.println("Failed to create directory: " + value);
+				}
+			    }
+
+			    // Set the file path as value + "output1.pem"
+			    file = value + "output1.pem";
+			} else {
+			    System.out.println("Output Path option not provided");
+			    // Set a default file path if "path" option is not provided
+			    String value = "/home/prerana/Desktop/PCVT/PCVT/src/main/java/hwManifestGen/";
```
Comment (gustavobbrand) [inline src/main/java/hwManifestGen/AllComponents.java]: At the instructions the paths suggested were in general /opt/hpe/scl/ or /opt/hpe/scl/certificates for the outputs... so I suggest to modify this for /opt/hpe/scl/ or a directory whithin this path.

**Review by gustavobbrand — CHANGES_REQUESTED** [review]
Looks good to me in general, but please check the comments regarding the fixed paths.

File: src/main/java/hwManifestGen/AllComponents.java (hunk 1 above)
Comment (preranam05) [reply src/main/java/hwManifestGen/AllComponents.java]: Changed to the expected path

File: src/main/java/hwManifestGen/AllComponents.java (hunk 2 above)
Comment (preranam05) [reply src/main/java/hwManifestGen/AllComponents.java]: Changed to the expected path

**Pull request closed without merging**